packages/commonwealth/client/scripts/views/pages/CommunityGroupsAndMembers/Groups/common/GroupForm/TopicPermissionsSubForm/TopicPermissionsSubForm.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { CWText } from 'views/components/component_kit/cw_text';
+import { CWSelectList } from 'views/components/component_kit/new_designs/CWSelectList';
+import { TOPIC_PERMISSIONS } from '../constants';
+import { TopicPermissionsSubFormType } from '../index.types';
+import './TopicPermissionsSubForm.scss';
+
+const permissionMap = Object.values(TOPIC_PERMISSIONS).map((permission) => ({
```

Comment:
a dropdown with combined permissions feels kind of weird... a list with checkboxes to allow granular selection is more flexible and intuitive IMO

Reply:
Seems like the design team had some thoughts about that and made some initial blueprints [here](https://www.figma.com/design/u9xVULF2IzBU4riycghQil/Gating-v2?node-id=773-2877&node-type=frame&t=3llHOMBHvMG0d1zg-0) but eventually settled on the dropdown design [here](https://www.figma.com/design/u9xVULF2IzBU4riycghQil/Gating-v2?node-id=773-4078&node-type=frame&t=3llHOMBHvMG0d1zg-0).

I agree the checkboxes/toggles offer more granular control, but I think they will get overwhelming ex: `4x dropdowns for each topic, and we have 4 topics = 16 checkboxes + the mobile screen size adjustments would be needed`, the dropdown seems simpler in this case, but I will leave it to the design team.

cc: @sssssabinaaa @sachben91